src/sso/pages/AppDetails.tsx

```diff
@@ -88,7 +112,12 @@ const AppDetails = () => {
           </div>
 
           <div style={styles.logout}>
-            <TButton startIcon={<LogoutIcon></LogoutIcon>}>Logout</TButton>
+            <TButton
+              startIcon={<LogoutIcon></LogoutIcon>}
```

Comment:
please avoid uusing MUI as we are using paper

Reply:
I didn't add this.. this icon was already there i just added to logout function

Reply:
the icons are coming from MUI @Rebal67 I thought. The components like buttons are coming from Material paper